### PR TITLE
Expanded example, added missing quotes.

### PR DESCRIPTION
**Pull request by rajinder-yadav**

### CODE REVIEW

File: handout/directives/ng_style_directive.md [hunk 1]
```diff
@@ -24,3 +24,27 @@ export class StyleExampleComponent {
...
+    <p style="padding: 1rem"
+        [ngStyle]="alertStyles">
+        <ng-content></ng-content>
+    </p>
+  `
+})
+export class StyleExampleComponent {
+  borderStyle: string = '1px solid black';
+
+  // Must be JSON valid.
```
Comment (mhazy) [inline handout/directives/ng_style_directive.md]: I'd remove this comment, it's a plain object. The property values can also be functions that return values.

Comment (rajinder-yadav) [inline handout/directives/ng_style_directive.md]: Thanks

Comment (rajinder-yadav) [inline handout/directives/ng_style_directive.md]: Let me think of how to reword this comment!

File: handout/directives/ng_style_directive.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ We can remove the style properties out of the template into the component as a p
 export class StyleExampleComponent {
   borderStyle: string = '1px solid black';
 
-  // Must be JSON valid.
+  // Properties must be quoted.
```
Comment (mhazy) [inline handout/directives/ng_style_directive.md]: It just has to be enumerable object. Quotes are only required if it's a plain 'ol object and the CSS property contains a dash, which would not be valid, e.g. `border-radius`, in which case you may use camel case property values, e.g. `borderRadius`.

Here's some different ways to use ngStyle (e.g. using a class with properties defined programatically)

https://plnkr.co/edit/bDTs3s?p=preview

**Comment by bennett000** [comment]
LGTM :shipit:

**Pull request merged**